spec/features/admin/filter_products_spec.rb

```diff
@@ -1,6 +1,11 @@
-require "spec_helper"
+require 'spec_helper'
+require 'support/multiselect_helpers'
 
-describe "Filter products", :js do
+RSpec.configure do |c|
+  c.include MultiselectHelpers
+end
```

Comment:
What is the right way to do helpers such as these?

Reply:
See [spec_helper.rb](https://github.com/LocalOrbit/localorbit/blob/97a8d459925a571e9809522adef48196abc0edb7/spec/spec_helper.rb#L83)

Reply:
Have fixed this. Thanks.